apps/hashdotdev/src/_pages/blog/0001_block-design.mdx

```diff
@@ -34,7 +34,7 @@ Thankfully, you can avoid most of these usability problems by following our visu
 
 ## Styling Block Protocol blocks
 
-We've set up our [block starter template](https://github.com/blockprotocol/blockprotocol/tree/main/packages/block-template) with a couple of helpful CSS files:
+We've set up our [block starter template](https://github.com/blockprotocol/blockprotocol/tree/612431a4168b419f6843e254c6afa3c80fc55ede/packages/block-template) with a couple of helpful CSS files:
```

Comment:
Last commit before template was split in https://github.com/blockprotocol/blockprotocol/pull/480

Reply:
Don't we want to link to the individual latest versions of the `block-template-html`, `block-template-react` and `block-template-custom-element` templates here, rather than a frozen-in-time old version of our deprecated `block-template`?

Reply:
I could only find CSS variables like `--neutral-0: #ffffff;` etc. in `block-template-react`. This doc is an RFC so I've decided to point to a location that’s accurate at the time of writing.

Reply:
I don't see the point in updating something outdated and incorrect to something newer but equally wrong.

Reply:
How do you imagine the diff? Happy to update the PR accordingly.

Reply:
I think this is alright as at least the link works now and the point can be understood – it points to old code, but the blog post itself is old (relatively).

Reply:
I imagine our blog posts being evergreen, and attempts being made to keep them accurate. With "**Update:**" disclaimers at the top/bottom of posts if there is no point making minor rewrites or link updates to keep them accurate.

In this case, the block starter template doesn't appear to have been set up properly with the helpful CSS files we reference above.

We should add these files and ensure the blog post isn't misleading people. That is the fix/change we want here.

I suggest this PR be considered blocked until the _style service_ is drafted now, though, as the shape of these files (and perhaps approach taken as a whole) are likely to change.

Reply:
The link currently 404s though – surely it's better that it goes to somewhere for now, and we can update this blog in light of the styling service later?